cluster-autoscaler/estimator/binpacking_estimator.go

```diff
@@ -30,28 +28,25 @@ import (
 	schedulerframework "k8s.io/kubernetes/pkg/scheduler/framework"
 )
 
-// podInfo contains Pod and score that corresponds to how important it is to handle the pod first.
-type podInfo struct {
-	score float64
-	pod   *apiv1.Pod
-}
-
 // BinpackingNodeEstimator estimates the number of needed nodes to handle the given amount of pods.
 type BinpackingNodeEstimator struct {
-	predicateChecker predicatechecker.PredicateChecker
-	clusterSnapshot  clustersnapshot.ClusterSnapshot
-	limiter          EstimationLimiter
+	predicateChecker     predicatechecker.PredicateChecker
+	clusterSnapshot      clustersnapshot.ClusterSnapshot
+	limiter              EstimationLimiter
+	podPriorityProcessor PodPriorityProcessor
 }
 
 // NewBinpackingNodeEstimator builds a new BinpackingNodeEstimator.
 func NewBinpackingNodeEstimator(
 	predicateChecker predicatechecker.PredicateChecker,
 	clusterSnapshot clustersnapshot.ClusterSnapshot,
-	limiter EstimationLimiter) *BinpackingNodeEstimator {
+	limiter EstimationLimiter,
+	podPriorityProcessor PodPriorityProcessor) *BinpackingNodeEstimator {
 	return &BinpackingNodeEstimator{
-		predicateChecker: predicateChecker,
-		clusterSnapshot:  clusterSnapshot,
-		limiter:          limiter,
+		predicateChecker:     predicateChecker,
+		clusterSnapshot:      clusterSnapshot,
+		limiter:              limiter,
+		podPriorityProcessor: podPriorityProcessor,
 	}
 }
 
```

Comment:
The Estimate comment got outdated, IMO it should be updated with:
* Estimate only implements First-Fit binpacking, and the order of processing pods depends on the pod orderer.
* The default orderer is "Decreasing", and then the rest about First-Fit-Decreasing can be left after that.

Reply:
Done.